# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Determines when to begin user tracking
# goal message
int32 capture			# 0 = On, 1 = Off
